TodoApp.ConsoleApp/UI/TaskOperations.cs

```diff
@@ -0,0 +1,120 @@
+﻿using System;
+using System.Collections.Generic;
+using TodoApp.Library.Models;
+using TodoApp.Library.Data;
+using System.IO;
+
+
+namespace TodoApp.ConsoleApp.UI
+{
+    public static class TaskOperations
+    {
+        public static void ReadOrWrite()
+        {
+            bool choice = true;
+
+            while (choice == true)
+            {
+                int operation = UserInput.ReadOption("Choose an option!", new string[] { "manipulate task", "read task's" }, true);
+
+                switch (operation)
+                {
+                    case 1:
+                        ManipulateTask();
+                        break;
+                    case 2:
+                        ReadTasks();
+                        break;
+                    default:
+                        break;
```

Comment:
`default` that does not do anything is unnecessary.
You can add a message to the user "Error! Invalid command!".